agent/consul/acl.go

```diff
@@ -211,6 +212,40 @@ type ACLResolverConfig struct {
 	Tokens *token.Store
 }
 
+// TODO: rename the fields to remove the ACL prefix
+type ACLResolverSettings struct {
```

Comment:
Probably best to wait until this moves elsewhere to bother renaming. Then you'll have the package name to help prefix it nicely.